r-reproducibility.Rmd

```diff
@@ -0,0 +1,1211 @@
+```
+
+which gives...
+
+![Steps to being more reproducible. Source DOI: [10.1038/d41586-018-05990-5](https://doi.org/10.1038/d41586-018-05990-5)](figures/r-reproducibility/code-sharing-steps.png)
+
+*Tip*: Can also include links to images from the Internet, as a URL link.
+
+If you want to modify the width or sizing, append something like `{width=##%}` to 
+the end of the image insertion:
+
+```markdown
+![Caption.](figures/r-reproducibility/code-sharing-steps.png){width=50%}
+```
+
+which gives...
+
+![Caption.](figures/r-reproducibility/code-sharing-steps.png){width=50%}
+
+#### Adding links to websites
+
+And a link can be linked in the following format:
+
+```markdown
+[Link to GitHub](https://github.com).
+```
+
+gives...
+
+[Link to GitHub](https://github.com).
+
+The above form is great for one time use, but what if you want to use the same link
+again later on? Use this form then:
+
+```markdown
+Use multiple [links] in your document. The same [links] can be used again.
+
+[links]: https://github.com
+```
+
+which gives...
+
+Use multiple [links] in your document. The same [links] can be used again.
+
+[links]: https://github.com
+
+#### Inserting (simple) tables
+
+You can insert tables with Markdown too. We wouldn't recommend doing it for 
+complicated tables though, as it can get tedious fast! (A recommended approach
+for more complex or bigger tables is to make the table contents as a data frame
+in R first and then use the `knitr::kable()` function to create the table, as
+we'll cover in the R Markdown section below). You can even include Markdown
+text formatting inside the table:
+
+```markdown
+|   | Fun | Serious |
+|:--|--:|:--:|
+| **Happy** | 1234 | 5678 |
+| **Sad** | 123 | 456 |
+```
+
+which gives...
+
+|   | Fun | Serious |
+|:--|--:|:--:|
+| **Happy** | 1234 | 5678 |
+| **Sad** | 123 | 456 |
+
+The `|--:|` or `|:--|` tell the table to right-align or left-align,
+respectively, the values in the table column. Center-align is `|:--:|`.
+See how it looks by "Knitting" the document ("Knit" button or `Ctrl-Shift-K`).
+
+#### Exercise: Try to re-create a document using Markdown
+
+1. Go to (link to html page). This is a HTML file that has been created by using
+Markdown formatting. 
+2. Create a new R Markdown file, with output type "HTML", and save it as
+`mimic-html-file.Rmd`.
+3. Delete all the automatically added text except the top part Inside the R Markdown file
+4. Write text using Markdown formatting so that you
+can create a `html_document` that looks exactly like that (linked) file.
+4. Knit the 
+
+### R Markdown
+
+[R Markdown][r-markdown] is an extension of Markdown that weaves together R code
+with Markdown formatted text all together in a single document. Output from the
+R code gets inserted directly into the document for a seamless integration of
+document writing and analysis reporting.
+
+#### YAML header/metadata
+
+Most Markdown documents (especially for R Markdown) include [YAML][yaml] metadata at
```

Comment:
What is "YAML"? Do I really need it? Why is this so hard? (Honestly, YAML seems to be the hardest part of R Markdown...)

Reply:
I've added some clarifications, let me know if that helps.